clients/web/src/lib/components/molecules/scrims/AdminBanTable.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+    import {activePlayers, type Player} from "$lib/api";
+    import {bannedPlayers, type MemberRestrictionEvent, cancelRestrictionMutation} from "$lib/api";
+
+    //export let visible = false;
+    let bannedPlayersData: MemberRestrictionEvent[] | undefined;
+    $: bannedPlayersData = $bannedPlayers?.data?.getActiveMemberRestrictions;
+
+    let targetRestriction:number;
+
+    const unbanThisPlayer = (restrictionId: number) => {
+        targetRestriction = restrictionId;
+        console.log(`Unbanning ${restrictionId}`);
+        cancelRestriction(restrictionId);
+    }
+
+    function cancelRestriction(id:number) {
+        try {
+            cancelRestrictionMutation({id: id?? 0, expiration: new Date()});
+        } catch {
+            console.log("oops, all berries!")
+        }
+    }
+</script>
+
+<table class="table text-center w-full" >
+    <thead>
+    <tr>
+<!--        <th>Ban Status</th>-->
+        <th>Player Name</th>
+        <th>Restriction ID</th>
+        <th></th>
+    </tr>
+    </thead>
+    <tbody>
+        {#if bannedPlayersData}
+            {#each bannedPlayersData as restriction}
```

Comment:
```suggestion
            {#each bannedPlayersData as restriction (restriction.id)}
```
This will add a "key" to the iteration which will make it faster, as well as handling updates to the dom more efficiently